File: app/src/main/java/com/babestudios/companieshouse/data/model/charges/Charges.java
package com.babestudios.companieshouse.data.model.charges;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Charges {
	@SerializedName("etag")
	public String etag;
	@SerializedName("items")
	public List<ChargesItem> items = new ArrayList<>();
	@SerializedName("part_satisfied_count")
	public String partSatisfiedCount;
	@SerializedName("satisfied_count")
	public String satisfiedCount;
	@SerializedName("total_count")
	public String totalCount;
	@SerializedName("unfiletered_count")
	public String unfileteredCount;
}

File: data/src/main/java/com/babestudios/companyinfo/data/model/charges/Charges.kt
package com.babestudios.companyinfo.data.model.charges

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class Charges {
	@SerializedName("etag")
	var etag: String? = null
	@SerializedName("items")
	var items: List<ChargesItem> = ArrayList()
	@SerializedName("part_satisfied_count")
	var partSatisfiedCount: String? = null
	@SerializedName("satisfied_count")
	var satisfiedCount: String? = null
	@SerializedName("total_count")
	var totalCount: String? = null
	@SerializedName("unfiletered_count")
	var unfileteredCount: String? = null
}
